### PR TITLE
fix(get-starknet): show popup ad when enabling without extension installed

### PR DESCRIPTION
**Pull request by delaaxe**

### CODE REVIEW

**Comment by janek26** [comment]
Not sure this change solves the issue, does it? For me it looks the same, except that the argument is passed later. But the logic isn't touched, therefore it shouldn't change the behavior of the popup, does it?

**If we decide to keep this we would need to change the Readme.md accordingly**

**Comment by delaaxe** [comment]
The logic is different because of the global state. I've updated the README

File: Readme.md [hunk 1]
```diff
@@ -51,13 +47,13 @@ The package is a light wrapper around [starknet.js](https://github.com/seanjames
 ```javascript
 import { getStarknet } from "@argent/get-starknet"
 
-// check if wallet extension is installed and initialized. Shows a modal prompting the user to download ArgentX otherwise.
-const starknet = getStarknet({ showModal: true })
-const [userWalletContractAddress] = await starknet.enable() // may throws when no extension is detected
+const starknet = getStarknet() // May throw when no extension is detected
+// Check if wallet extension is installed and initialized. Shows a modal prompting the user to download Argent X otherwise.
```
Comment (janek26) [inline Readme.md:51]: This makes no sense to me :D `getStarknet()`may throws when no extension is detected? why? shouldn't `enable()` throw (and conditionally show the popup) if no extension is detected?

Comment (delaaxe) [reply Readme.md]: yeah was a bit too fast on this one, fixed

**Review by janek26 — APPROVED** [review]

**Pull request merged**